src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--fri calendar__month-length--31">
+      <div class="calendar__cell calendar__cell--day--1"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
Тебе не нужен тут дополнительный класс, просто first-child стилизуй по наличию класса у calendar